tests/public-api/test_solvers_jobs_api.py

```diff
@@ -169,6 +179,17 @@ def test_create_job(
     assert job.id != job2.id
 
 
+_RETRY_POLICY_IF_LOGFILE_404_NOT_FOUND = dict(
+    # NOTE: Only 404s https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+    # are retried, the rest are failures
+    retry=retry_if_exception_type(TryAgain),
+    wait=wait_fixed(1),
+    stop=stop_after_attempt(3),
```

Comment:
3 seconds for the file to get there seems a bit low to me. I'd expect this still to fail. What about giving it more total time?

Reply:
I think waiting 3 seconds is reasonable for such a small file. If it takes longer it might highlight other defects. I would take it as a minimal requirement, observe and see how it evolves